### ISSUE TITLE
Add information about royalties on design claim

### ISSUE THREAD

**Issue by devey** [devey/YSN#5]
```ts
@nearBindgen
export class Royalty {
    split_between: Map<AccountId, u32> = new Map();
    percentage: u32 = 0;
    constructor() {
        /** 25% of future sales goes to FT */
        this.split_between.set(FT_CONTRACT, FT_CONTRACT_ROYALTY);
        this.percentage = FT_CONTRACT_ROYALTY;
    }
}

```

`Royalties` in the `SHARE` contract are currently enforced to give a perpetual `25%` to the `YSN` contract.
Need to let that known in the front-end.

**Comment by devey** [comment]
Closed in #6

**Comment by devey** [comment]
Closed in #6
